designs/virtualization/4.7/add-ip-addresses/add-ip-addresses.md

```diff
@@ -0,0 +1,28 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Add V4 and V6 IP Addresses
+
+## Adding a ‘Last restored’ column to the list view
+
+Currently we only show IPv4 IP addresses but with the addition of IPv6 we should consider how these strings should surface in the list/details view of the virtual machine.
+Currently we would not be able to show both types together though.
+
+## List view
+
+![List view](img/VM-list.png)
```

Comment:
can we also point out the change in the column header?

Reply:
+1